draft-ietf-acme-acme.md

```diff
@@ -543,7 +543,6 @@ in the "type" field (within the "urn:ietf:params:acme:error:" namespace):
 | rateLimited             | The request exceeds a rate limit                                               |
 | rejectedIdentifier      | The server will not issue for the identifier                                   |
 | serverInternal          | The server experienced an internal error                                       |
-| tls                     | The server received a TLS error during validation                              |
```

Comment:
TLS errors can still happen if HTTP-01 redirects to https:// and TCP connection to port 443 succeeds, but the TLS handshake fails.